### PR TITLE
Error messages

### PR DESCRIPTION
**Pull request by Poeschl**
I implemented some logic to warn the user about the requirements of the play store. #64 
So now you will get an error if the texts exceeds the given limits.
For more look at the Updated README (7d2d02cd0dc2d099506f50476be375e0be396666).

### CODE REVIEW

**Comment by bhurling** [comment]
Nice work

**Comment by bhurling** [comment]
Fixes #64

File: src/main/groovy/de/triplet/gradle/play/PlayPublishListingTask.groovy [hunk 1]
```diff
@@ -41,15 +41,28 @@ class PlayPublishListingTask extends PlayPublishTask {
...
+                    if (!TaskHelper.checkForTextLength(fileTitle, MAX_CHARACTER_LENGTH_FOR_TITLE)) {
+                        throw new LimitExceededException(fileTitle, MAX_CHARACTER_LENGTH_FOR_TITLE)
+                    }
+                    if (!TaskHelper.checkForTextLength(fileShortDescription, MAX_CHARACTER_LENGTH_FOR_SHORT_DESCRIPTION)) {
+                        throw new LimitExceededException(fileShortDescription, MAX_CHARACTER_LENGTH_FOR_SHORT_DESCRIPTION)
+                    }
+                    if (!TaskHelper.checkForTextLength(fileFullDescription, MAX_CHARACTER_LENGTH_FOR_FULL_DESCRIPTION)) {
+                        throw new LimitExceededException(fileFullDescription, MAX_CHARACTER_LENGTH_FOR_FULL_DESCRIPTION)
+                    }
+                    if (!TaskHelper.checkForTextLength(fileVideo, Integer.MAX_VALUE)) {
```
Comment (ChristianBecker) [inline src/main/groovy/de/triplet/gradle/play/PlayPublishListingTask.groovy]: Is this check useful?

**Comment by bhurling** [comment]
I think it might be even smarter to move that check into the `TaskHelper.readAndTrimFile()` itself. In fact, there is already a check for that. We just have to pass that boolean with it.

**Comment by bhurling** [comment]
The property itself is very helpful. Without it we would just cut off characters without telling the developer.

**Comment by ChristianBecker** [comment]
I was just curious to check for `Integer.MAX_VALUE` ;) Ok, I'll move this check into the `readAndTrimFile()` method in a follow-up.

**Pull request merged**